src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -213,7 +213,8 @@ public without sharing class UTIL_OrgTelemetry_SVC {
      */
     private Integer getCountOfRecurringDonationOpps() {
         try {
-            return [SELECT Count() FROM Opportunity WHERE npe03__Recurring_Donation__c != null];
+            return [SELECT Count() FROM Opportunity
```

Comment:
Can we have a comment indicating the query was written in this specific way to work on LDV orgs?